src/components/Reports/ReportEdit.tsx

```diff
@@ -144,10 +144,22 @@ export default function ReportEdit({ organization }: ReportEditProps): JSX.Eleme
     }
   };
 
+  const goToTop = () => {
+    window.scrollTo({
+      top: 0,
+      behavior: 'smooth',
+    });
+  };
+
+  const switchPages = (doShowAnnual: boolean) => {
+    setShowAnnual(doShowAnnual);
+    goToTop();
```

Comment:
do we need a setTimeout of 0 on the scroll .. wait until setShowAnnual does it re-rendering?

Reply:
It seems to render before scrolling already. I think this is okay